app/config/sources.inc.php

```diff
@@ -376,9 +377,8 @@
     'snippets/2016/dec2016.lang' => [
         'critical' => ['de', 'es', 'fr', 'pt-BR', 'ru'],
     ],
-    'snippets/2016/dec2016_eoy_a.lang' => [ 'critical' => ['all'] ],
```

Comment:
Not really, if you run the script for untracked files you would end up removing them.